### ISSUE TITLE
Does it work for mobile phones?

### ISSUE THREAD

**Issue by ferigis** [fcevado/phone#138]
hi!

Thanks for your library, it seems great :) 

I was trying to parse mobile phones from Spain, but it doesn't work (I think)

```elixir
Phone.parse "+34600111111"
{:error, "Not a valid phone number."}
```

Is that expected and only work for landlines?

Thanks!

**Comment by fcevado** [comment]
it should work for cellphones too. recently support for regional codes for spanish numbers were added. I'll need to take a look at the regexes there. @ferigis if you know how to fix it, i'm glad to merge your changes.
can you point me to the spanish numbering plan in english? it would make it easier.

**Comment by LuhalvesBR** [comment]
Hello lads,
I am just seeing this problem... It looks like the code is not treating the mobile phone in 🇪🇸 
(My bad, the change I had sent previously didn’t include the format @fcevado )

@ferigis, correct if I am getting it wrong but the format for mobile number in Spain should be:

```
+34 6XY...
+34 71Y...
+34 72Y...
+34 73Y...
+34 74Y...
(X and Y can be any digit from 0 to 9)
```
So, we are gonna have this format:
<img src="https://user-images.githubusercontent.com/11468657/109527456-f6021b80-7ab3-11eb-8506-fbf58a2cbda9.png" width="40%" height="40%">

Any suggestions on how to return the following keys?
area_abbreviation, area_code, area_name, area_type

I can send a PR with this small change soon 🔥
Thanks, y'all!

**Comment by fcevado** [comment]
@LuhalvesBR a simple way to solve it is to create a `Phone.ES.Mobile` as the way it's done for toll free numbers for Brazil or NANP. 
just an example:
https://github.com/fcevado/phone/blob/main/lib/phone/br/toll_free.ex

**Comment by fcevado** [comment]
fixed by #140
